pom.xml

```diff
@@ -131,7 +131,6 @@
       <dependency>
```

Comment:
Since gson is dependency managed in shared-deps, removing the whole dependency from here should be fine.

Reply:
Removed